src/Stratis.Bitcoin.Features.SmartContracts.Tests/Stratis.Bitcoin.Features.SmartContracts.Tests.csproj

```diff
@@ -18,7 +18,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
-    <PackageReference Include="Stratis.SmartContracts" Version="1.2.1" />
+    <PackageReference Include="Stratis.SmartContracts" Version="1.4.1-alpha" />
```

Comment:
These version needs to be inline with FN (1.0.6.4)

Reply:
Previously it seems the SCs had their own version sequence. Even Stratis.SmartContracts and Stratis.SmartContracts.Standads had different versions. I guess the advantage was the ability to release an update of just one module and not worry about the others being different. Are you sure we should try and keep versions in step?